src/components/NavBar/index.tsx

```diff
@@ -5,14 +5,17 @@ import { HiOutlineShoppingBag } from "react-icons/hi";
 import { IoIosArrowBack } from "react-icons/io";
 import { RiArrowGoBackLine, RiArrowGoForwardLine } from "react-icons/ri";
 import { BiStar, BiPencil } from "react-icons/bi";
+import Link from "next/link";
 
 const NavigationBar = () => {
   return (
     <Grid templateColumns="repeat(3, 1fr)" bg="brand.primary" p={4}>
       <Flex alignItems="center">
-        <Button leftIcon={<IoIosArrowBack />} pl="0" variant="navbarIconBtn">
-          Home
-        </Button>
+        <Link href="https://courtcanva.com/" passHref>
```

Comment:
I suggest extracting this URL as a constant so that we can dynamically change it according to different environments.  If I remember correctly, this URL is of the landing page in the Production environment. We should keep all environments independent, so UAT or even other environments should have their own landing page instead of sharing the same one with PROD.